src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsParserStateObject.cs

```diff
@@ -2526,43 +2574,45 @@ internal bool IsConnectionAlive(bool throwOnException)
             return isAlive;
         }
 
-        /*
+        
 
         // leave this in. comes handy if you have to do Console.WriteLine style debugging ;)
```

Comment:
I've added a function description. I also changed the brace style to c# instead of java. This function is as old as the codebase, I didn't add it I just used it.